pkg/storage/memory/repository.go

```diff
@@ -0,0 +1,59 @@
+package memory
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hisshoes/crypto-rebalancer/pkg/portfolio"
+
+	uuid "github.com/nu7hatch/gouuid"
```

Comment:
Swap this and your own import around, positioning wise.

Reply:
As discussed previously, I'd recommend https://github.com/gofrs/uuid if you want uuid and I would recommend looking at https://github.com/segmentio/ksuid if you think you'd be interested in sortable unique IDs. Just as an option.